doc-content/enterprise-only/springboot/spring-boot-jms-audit-proc.adoc

```diff
@@ -0,0 +1,126 @@
+[id='spring-boot-jms-audit-proc_{context}']
+= Replicating audit data in a JMS message broker
+
+You can replicate {KIE_SERVER} audit data to a Java Message Service (JMS) message broker, for example ActivMQ or Artemis, and then dump the data in an external database schema so that you can improve the performance of your Spring Boot application by deleting the audit data from your application schema.
```

Comment:
Can you break up this long sentence? Also, made a couple of minor edit suggestions.

You can replicate {KIE_SERVER} audit data to a Java Message Service (JMS) message broker. For example, ActivMQ or Artemis, and then dump the data in to an external database schema to improve the performance of your Spring Boot application by deleting the audit data from your application schema.